modules/cluster-logging-collector-log-forward-gcp.adoc

```diff
@@ -25,38 +25,44 @@ Using this feature with Fluentd is not supported.
 ----
 $ oc -n openshift-logging create secret generic gcp-secret --from-file google-application-credentials.json=_<your_service_account_key_file.json>_
 ----
+
 . Create a `ClusterLogForwarder` Custom Resource YAML using the template below:
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: openshift-logging
 spec:
-  serviceAccountName: <service_account_name> <3>
+  serviceAccountName: 
+    name: <service_account_name> #<1>
   outputs:
     - name: gcp-1
       type: googleCloudLogging
-      secret:
-        name: gcp-secret
       googleCloudLogging:
-        projectId : "openshift-gce-devel" <4>
-        logId : "app-gcp" <5>
+        id:
+          type : project
+          value: openshift-gce-devel #<2>
+      logId : app-gcp #<3>
+      authentication:
+        credentials:
+          key: google-application-credentials.json
+          secretName: gcp-secret
   pipelines:
     - name: test-app
-      inputRefs: <6>
+    - inputRefs: #<4>
         - application
+      name: test-app
       outputRefs:
         - gcp-1
 ----
-<1> In legacy implementations, the CR name must be `instance`. In multi log forwarder implementations, you can use any name.
-<2> In legacy implementations, the CR namespace must be `openshift-logging`. In multi log forwarder implementations, you can use any namespace.
-<3> The name of your service account. The service account is only required in multi log forwarder implementations if the log forwarder is not deployed in the `openshift-logging` namespace.
-<4> Set a `projectId`, `folderId`, `organizationId`, or `billingAccountId` field and its corresponding value, depending on where you want to store your logs in the link:https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy[GCP resource hierarchy].
-<5> Set the value to add to the `logName` field of the link:https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry[Log Entry].
-<6> Specify which log types to forward by using the pipeline: `application`, `infrastructure`, or `audit`.
+<1> The name of your service account.
+<2> Set a `projectId`, `folderId`, `organizationId`, or `billingAccountId` field and its corresponding value, depending on where you want to store your logs in the link:https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy[GCP resource hierarchy].
+<3> Set the value to add to the `logName` field of the link:https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry[Log Entry].
+<4> Specify which log types to forward by using the pipeline: `application`, `infrastructure`, or `audit`.
```

Comment:
This again is technically not accurate.   The original text is probably very old, since we have allowed custom input types since early in v5.    The `inputRefs` is a reference of input `name`  (although we have reserved "default" values)